cli/command/service/logs.go

```diff
@@ -49,13 +49,14 @@ func newLogsCommand(dockerCli *command.DockerCli) *cobra.Command {
 			opts.target = args[0]
 			return runLogs(dockerCli, &opts)
 		},
-		Tags: map[string]string{"experimental": ""},
```

Comment:
Actually, just recalled that I opened an issue for (roughly) this https://github.com/docker/docker/issues/28655. If we agree on that one, it should probably be blocked for older versions